packages/node_modules/@webex/internal-plugin-ediscovery/src/ediscovery.js

```diff
@@ -28,9 +28,10 @@ const EDiscovery = WebexPlugin.extend({
   /**
    * Creates a compliance report with a specific set of search parameters
    * @param {Object} reportRequest - A set of criteria for determining the focus of the search
+   * @param {int} timeoutMs - optional connection timeout in milliseconds, defaults to 10s
    * @returns {Promise<ResponseEntity>} Http response containing the new report record
    */
-  createReport(reportRequest) {
+  createReport(reportRequest, timeoutMs = 10000) {
```

Comment:
Since this is an optional parameter, would it make more sense to wrap it in an options object as is done below for the paged APIs?

Reply:
updated